Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,33 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_environment",
+            category=['society'],
+            prompt="The environment",
+            random=True
+        )
+    )
+ 
+label monika_environment:
+    m 2eub "[player], do you know why I care about the environment?"
+    m 1esa "Well, there are a few reasons."
+    m 3euu "For one, I think it's beautiful."
+    m 1dsd "Vibrant meadows in the spring, chittering forests in the fall..."
+    m 2hub "Breathtaking beaches in the summer, and shimmering snow in the winter."
+    m 7wsu "Sometimes it can be euphoric just to sit down and admire nature, you know?"
+    m 1lut "The environment also sustains life."
+    m 4dsc "Biodiversity, decomposition, pollination, reduced erosion, heat absorption, good air and water quality..."  
+    m 6rkd "Without the environment, none of that would be possible, and life would have never existed as it does today."
+    m 1fkc "All Earth would have been is a dead rock with some microbes sprinkled here and there, and you and I would have never existed."
```

Comment:
![4](https://user-images.githubusercontent.com/45746485/93007443-7e721b80-f569-11ea-9904-40aa10b64d20.png)

I don't really like the soft eyes in these situations, she doesn't have to seem like she's about to cry because of a hypothetical situation. Suggestion:

![image](https://user-images.githubusercontent.com/45746485/93007500-e294df80-f569-11ea-9b78-e20d6f1f38d5.png)

Reply:
I see what you mean, but with limited choices I really want to try and make each pose as unique as possible to maximize all possible combinations (that includes trying to use soft eyes where normal eyes may look better). I don't know, I'm not sure about this one. I'll probably decide on it in the morning.

Reply:
You probably don't have to worry about it looking unnatural. Most of the game's topics re-use the most common expressions over and over again. The most important thing is that Monika's expression matches what she says, and her personality overall.

Reply:
The thing is, it's exactly that the game uses the same expressions over and over again that it feels unnatural; she ONLY uses those common expressions for the most part, which limits the diversity of her reactions and makes them feel limited (in my opinion). I try to balance Monika's expression with how she's feeling, but I also want to see if I can avoid using common expressions where I can since (to me) that would make it feel less artificial.

Reply:
Expressions should be picked as "most fitting for the line", not "most varied"

Reply:
I don't think it feels unnatural at all. It would feel unnatural if her expression never changed, not even slightly. I was looking over your other PR and noticed you were using a lot of rarer exp choices. 

If you ask me, this actually makes it look more unnatural because you have her making all these vibrant, outlandish faces, instead of well, talking like how you would talk in a typical conversation. Maybe it's just me, but when I'm talking, my face barely changes. It does, but only with slight, subtle changes.

Reply:
In day to day life, how many different expressions does a person really make? And people don't try to make different faces to mix it up, their face matches their mood, and it should be the same here.

Reply:
I've probably overcomplicated this, so I apologize. I do want to suggest weird poses where I can, but I understand that having variety solely for variety's sake would be problematic (unrealistic too, you all have a good point there). I'll shift my approach.